library/src/test/kotlin/de/adesso/junitinsights/spring/InsightConfigurationTest.kt

```diff
@@ -0,0 +1,32 @@
+package de.adesso.junitinsights.spring
+
+import com.nhaarman.mockitokotlin2.*
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Test
+import org.springframework.context.support.GenericApplicationContext
+
+class InsightConfigurationTest {
+
+    @Test
+    @Disabled
```

Comment:
Is this still WIP or did you forget to remove this?

Reply:
Yup, that's still WIP. Sorry for not marking it explicitly. Testing Spring is still kinda difficult for me to understand. We might need to scrub that idea and start over, but before we do that, I wanted to show what I tried. This also relates to the `open` thing from above